packages/components/src/components/contract-card/contract-card-items/vanilla-options-card-body.tsx

```diff
@@ -1,16 +1,34 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { getDisplayStatus, isCryptocurrency } from '@deriv/shared';
+import { TContractInfo } from '@deriv/shared/src/utils/contract/contract-types';
```

Comment:
we are still using this type instead of return type from usestore

Reply:
What's the issue of using the type here? I tried your solution, it doesn't seem to be working and we use this type everywhere else as well.

Reply:
shared type is incomplete and need to move away from it. Yes you'll have some typescript errors but should be fixed.

Reply:
nvm seems they updated the shared type from master. They are now identical, weird that mine would show an error now.